src/components/periodic-report/periodic-report.resolver.ts

```diff
@@ -53,11 +69,50 @@ export class PeriodicReportResolver {
     return await this.service.update(input, session);
   }
 
-  @ResolveField(() => SecuredFile)
+  @ResolveField(() => SecuredFile, {
+    description:
+      'Returns Pnp file for progress report. Otherwise returns first report file of directory',
+  })
```

Comment:
This can be deprecated. `directory` or `pnp` should be recommended instead. Speaking of which we need to expose `pnp` in a new `ProgressReportResolver`.